src/caselawclient/client_helpers/__init__.py

```diff
@@ -27,14 +29,25 @@ class VersionType(Enum):
 class VersionAnnotation:
     """A class holding structured data about the reason for a version."""
 
-    def __init__(self, version_type: VersionType, message: Optional[str] = None):
+    def __init__(
+        self,
+        version_type: VersionType,
```

Comment:
Appreciate it's not new code, but importing two things from client_helper (VersionAnnotation and VersionType) to craft an annotation feels like it's breaking a boundary. Should we expose these in client directly?